scene/level_one.py

```diff
@@ -33,6 +33,7 @@ def __init__(self):
 
         self.watermelon = Watermelon(seed_inventory=30)
         self.add_fg_element(self.watermelon)
+        camera.set_target(self.watermelon)
```

Comment:
It's cool we can set the camera target.  Might make for some interesting camera tricks like following a seed or fight a boss from their perspective.